synapse/datamodel.py

```diff
@@ -125,10 +126,11 @@ def __init__(self, load=True):
         self.addTufoProp('syn:form', 'ptype', ptype='syn:type', req=1, doc='Synapse type for this form')
 
         self.addTufoForm('syn:prop', ptype='syn:prop')
-        self.addTufoProp('syn:prop', 'doc', ptype='str', req=1, doc='Description of the property definition')
+        # TODO - Re-enable syn:prop:doc req = 1 after cleaning up property docstrings.
```

Comment:
I'd add another story to track adding docstrings for all props and enabling this. We should inform users before enabling this to avoid private models breaking.

Reply:
@therealsilence has a story for doing all these docs already.